src/routes/Route.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { BrowserRouter, Routes, Route } from "react";
+
+const Route = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element="home" />
+        <Route path="master-banner" element="" />
+      </Routes>
+    </BrowserRouter>
+  );
+};
+
+export default Route;
```

Comment:
this is old tradition, we have moved to `react-router-dom` `v6` so try to implement that way. 
link : https://reactrouter.com/en/main/start/overview